pyiron_base/database/filetable.py

```diff
@@ -47,13 +48,26 @@
 }
 
 
-class FileTable(IsDatabase, metaclass=Singleton):
-    def __init__(self, project):
-        self._fileindex = None
-        self._job_table = None
-        self._project = os.path.abspath(project)
-        self._columns = list(table_columns.keys())
-        self.force_reset()
+class FileTable(IsDatabase, metaclass=UpdatableSingleton):
+    def __init__(self, path: Optional[str] = None):
+        try:
+            if path is not None and os.path.abspath(path) != self._project:
```

Comment:
I feel this is dangerous. The typical example that comes to mind is a project with path `a/b/c` followed by a subproject with path `a/b/c/d` .

Reply:
Isn't it just "dangerous" in the sense that you may pay the computational cost of re-indexing when you don't need to?

IMO the fact this thing is still a singleton _at all_ is what's dangerous.

I opened another PR (#1075) which follows your other advice of just making new instances.

I'd like to continue discussion of both on the issue #1073